plugins/services/src/js/containers/service-debug/ServiceDebugContainer.js

```diff
@@ -83,6 +98,81 @@ class ServiceDebugContainer extends React.Component {
     return <DescriptionList hash={LastVersionChangeValueMapping} />;
   }
 
+  getRecentOfferSummary() {
+    const {service} = this.props;
+    const queue = service.getQueue();
+    let introText = null;
+    let mainContent = null;
+    let offerCount = null;
+
+    if (this.isFramework(service)) {
+      const {labels = {}} = service;
+      const frameworkName = labels.DCOS_PACKAGE_FRAMEWORK_NAME;
+
+      if (frameworkName != null) {
+        introText = `Rejected offer analysis is not currently supported for ${frameworkName}.`;
+      } else {
+        introText = 'Rejected offer analysis is not currently supported.';
+      }
+    } else if (queue == null || queue.declinedOffers.summary == null) {
+      introText = 'Offers will appear here when your service is deploying or waiting for resources.';
+    } else {
+      const {declinedOffers: {summary}} = queue;
+      const {roles: {offers = 0}} = summary;
+      const docsURL = MetadataStore.buildDocsURI(
+        '/overview/concepts/#mesos-resource-offer'
+      );
+
+      introText = (
+        <span>
+          When you attempt to deploy a service, DC/OS waits for offers to match the resources your service requires. If the offer does not satisfy the requirement, it is declined and DC/OS retries. <a href={docsURL} target="_blank">Learn more</a>.
```

Comment:
This block of text is repeated exactly in both Pods and Services debug container, reckon it's worth moving it to a constant somewhere?